src/styles/atoms/_atoms.table.scss

```diff
@@ -111,21 +112,59 @@
   user-select: none;
   width: 100%;
 
-  .ai {
-    color: var(--table-sort-btn-icon-color);
-    font-size: .875rem;
-    margin-left: var(--spacer-3xs);
-  }
-
   &:hover,
   &:active,
   &:focus {
-    background-color: var(--table-sort-btn-hover-bg-color);
+    --table-sort-btn-bg-color: var(--table-sort-btn-hover-bg-color);
   }
 
   &.a-table__sort-button--align-right {
     justify-content: flex-end;
   }
+
+  &.a-table__sort-button--unsorted,
+  &.a-table__sort-button--sorted-asc,
+  &.a-table__sort-button--sorted-desc {
+    --table-sort-btn-color: var(--table-sort-btn-active-color);
+
+    &:after {
+      background: var(--table-sort-btn-icon-color);
+      content: '';
+      height: 1rem;
+      margin-left: var(--spacer-3xs);
+      mask-size: contain;
+      width: 1rem;
+    }
+  }
+
+  &.a-table__sort-button--unsorted {
+    &:after {
+      mask: url('data:image/svg+xml;utf8,<svg id=%22expanded-vertical-1%22 viewBox=%220 0 17 17%22 xml:space=%22preserve%22 xmlns=%22http://www.w3.org/2000/svg%22><style>.st0{fill:none;stroke:%23000000;stroke-linecap:round;stroke-linejoin:round;}</style><g><path class=%22st0%22 d=%22M8.5,1.5v14%22/><path class=%22st0%22 d=%22M5.5,4.5l3-3l3,3%22/><path class=%22st0%22 d=%22M5.5,12.5l3,3l3-3%22/></g></svg>') no-repeat center center;
```

Comment:
Heb je dit in andere browsers getest? Ik weet dat er wel eens problemen zijn met bepaalde tekens, vandaar de vraag.